src/portal/src/app/base/project/project-config/project-policy-config/project-policy-config.component.scss

```diff
@@ -1,6 +1,9 @@
 #severity-blk div {
   display: inline-block;
 }
+.mr-10 {
```

Comment:
The current UI change is only limit to create project, why change the CSS of the project policy config?

Reply:
> The current UI change is only limit to create project, why change the CSS of the project policy config?

It was initially told by @zyyw that we need to make changes to both the create project and also to the  project policy-config page